src/fn/edit-contacts.js

```diff
@@ -9,13 +9,32 @@ const toDocs = require('./csv-to-docs');
 const EDIT_RESERVED_COL_NAMES = [ 'parent', '_id', 'name', 'reported_date' ];
 const DOC_TYPES = ['district_hospital', 'health_center', 'clinic', 'person', 'user', 'user-settings', 'contact'];
 const DOCUMENT_ID =  'documentID';
+const userPrompt = require('../lib/user-prompt');
 
 const execute = () => {
   const args = parseExtraArgs(environment.pathToProject, environment.extraArgs);
   const db = pouch();
   const docDirectoryPath = args.docDirectoryPath;
   fs.mkdir(docDirectoryPath);
-  const saveJsonDoc = doc => fs.write(`${docDirectoryPath}/${doc._id}.doc.json`, safeStringify(doc) + '\n');
+  let overwriteAllFiles = false;
+
+  const saveJsonDoc = doc => {
+    if(fs.exists(`${docDirectoryPath}/${doc._id}.doc.json`) && !args.updateOfflineDocs && !args.force && !overwriteAllFiles) {
```

Comment:
I think the code would be much easier to follow if we simplify this condition and put the "simple" action up top: 

So something like: 
```
if (args.updateOfflineDocs || args.force || overwriteAllFiles) {
  return writeFile(doc);
}

if (!fs.exists(filepath)) {
   return writeFile(doc);
}

const userSelection = userPrompt.keyInSelect();
if (userSelection === undefined) {
  throw error; 
}

if (userSelection === 1) {
  overwriteAllFiles = true;
}
return writeFile(doc);
````